content/nextflow-on-aws-batch/35_nextflow201/11_job_queue.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Create Job Queues"
+chapter: false
+weight: 11
+---
+
+## Create Job Queue
+
+Let us created a queue via the console.
+![5_queue_workflow_0](/images/nextflow-on-aws-batch/batch/5_queue_workflow_0.png)
+
+
+To create the job queue we choose a name (**job-queue**) a priority of 1 and the map them to the correct compute environment (**spot-ce**).
```

Comment:
Change "priority of 1" to bold.

The goal is to let the attendees know in one glance where a keyboard input will be necessary.

Reply:
done